packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -3845,7 +3802,7 @@ export class ContainerRuntime
 		};
 	}
 
-	private async closeStaleSummarizer(codePath: string): Promise<void> {
+	private async closeStaleSummarizer(): Promise<void> {
```

Comment:
Why ditch the codePath param?  Oh, I guess it isn't used. I assumed it would have been logged. Maybe we can conclude which codePath it was from other logs anyway, not sure?

Reply:
It's unused. There was an event logged here but it was removed because there were too many events.